### PR TITLE
Remedy SR  - new integration

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
not ready - waiting for review with another customer

## Description
SOAP API calls implemented for create and get ticket commands for Remedy SR, with all relevant args, outputs, HR and response validations.

## Related Issues
fixes: https://github.com/demisto/etc/issues/16010

### open issues: integration name, logo  id, from\to version, command names, implement update, xml_ns parameter.

print screen and details from customer:
`For the Soap API version:     
Service Assurance
Service Request Service         
SOAP Service version 1.6
For the remedy version, it is TTS:`

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - added test pb. it won't run since we have no instance.
- [ ] Documentation - soon
- [x] Code Review

### CODE REVIEW

File: Tests/secrets_white_list.json [hunk 1]
```diff
@@ -717,6 +717,7 @@
   "generic_strings":
   [
     "self.conf_json_validator",
+    "http://xml.",
```
Comment (glicht) [inline Tests/secrets_white_list.json]: doesn't seem to be needed

File: Integrations/integration-remedy_SR.yml [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+      required: true
+      description: User City
+    - name: requester_work_location
+      required: true
+      description: User Office
+    - name: requester_work_street
+      required: true
+      description: User Street
+    description: Create a ticket
+  dockerimage: demisto/zeep:1.0.0.78
```
Comment (glicht) [inline Integrations/integration-remedy_SR.yml]: If not used. Remove

File: Integrations/integration-remedy_SR.yml [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+    import random
+    import requests
+
+    if not demisto.params().get('proxy', False):
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    VERIFY_CERTIFICATE = not demisto.params().get('insecure', True)
```
Comment (glicht) [inline Integrations/integration-remedy_SR.yml]: I think default should be False

**Comment by bakatzir** [comment]
Please note that i've added a test and updated conf.json accordingly - skipped the integration since we don't have an instance.

**Comment by bakatzir** [comment]
@yuvalbenshalom 
I fixed_ name,logo,description
need to go over the command names, params, and mandatory/default arguments for the commands.

**Comment by glicht** [comment]
approved the changes. please also wait for @yuvalbenshalom approval

**Pull request merged**